
package com.sms.common.response;

public class ApiResponse<T> {
    private boolean success;
    private String message;
    private T data;
}
